Slim/Plugin/SpotifyLogi/Plugin.pm

```diff
@@ -86,7 +86,9 @@ sub initPlugin {
 	);
 	
 	# in LMS we're going to authenticate every player - don't do this on mysb, it's too expensive
-	Slim::Plugin::SpotifyLogi::ProtocolHandler->init();
```

Comment:
This shouldn't be part of your pull request... Make sure only the text files (and others you might have changed) are part of it.